USAGE.md

```diff
@@ -57,7 +57,11 @@ OED_LOG_FILE=?                 // Path to the log file, defaults to ./log.txt
 1. Run ```systemctl enable oed.service``` to make the service start on server boot.
 1. Bring the app online with ```systemctl start oed.service```. Stop the app with ```systemctl stop oed.service```.
 
-### Administration ###
```

Comment:
You changed this file to USAGE.md but forgot to update the link in the README

Reply:
Fixed